app/src/main/java/com/easyfitness/fonte/FontesFragment.java

```diff
@@ -509,6 +508,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         mDisplayType = DisplayType.fromInteger(getArguments().getInt("displayType", DisplayType.FREE_WORKOUT_DISPLAY.ordinal()));
 
         machineEdit = view.findViewById(R.id.editMachine);
+        machineEdit.setSelectAllOnFocus(true);
```

Comment:
Couldn't this be done in the tab_fonte layout so that it is more coherent with the rest of the project code?

Reply:
I may have misinterpreted https://stackoverflow.com/questions/6392578/android-select-all-text-on-focus-on-editfield 
-> the xml attribute did not work in some cases (which should not be the case here as there are no focus grabbing elements)